docs/assistant/security-assistant.asciidoc

```diff
@@ -2,4 +2,97 @@
+[sidebar]
+--
+* The Elastic Security Assistant is available in {elastic-sec} version 8.8.1 and later.
+
+* This feature requires an https://www.elastic.co/pricing[Enterprise subscription].
+
+* You must have an account with a third-party generative AI provider, which the Security Assistant uses to generate responses. Supported providers are OpenAI (`gpt-3.5-turbo` model) and Azure OpenAI Service (any model).
+--
+
+[discrete]
+[[set-up-ai-assistant]]
+== Set up the Security Assistant
+
+You must complete these steps before you can use the Security Assistant:
+
+. Create an API key with your AI provider to authenticate requests from the Security Assistant. You'll use this in a later step. Refer to the provider's documentation on generating API keys:
++
+* https://platform.openai.com/docs/api-reference[OpenAI]
+* https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference[Azure OpenAI Service]
+
+. Add the following feature flag to {kib}'s configuration settings:
++
+`xpack.securitySolution.enableExperimental: ['assistantEnabled']` 
++
+The configuration method depends on your deployment type:
++
+* *Self-managed (on-premises) deployments*: Add the feature flag to the `kibana.yml` file, which is used to {kibana-ref}/settings.html[configure {kib}], then restart {kib}.
+* *{ecloud} deployments*: Use the YAML editor in the {ecloud} console to add the feature flag to {cloud}/ec-manage-kibana-settings.html[{kib} user settings].
+
+. Create a {kibana-ref}/gen-ai-action-type.html[Generative AI connector], using the AI provider's API key and URL to configure communication between {elastic-sec} and the provider. You can do this in {kib} at *Stack Management* -> *Connectors*, or from within the Security Assistant.
++
+NOTE: The Generative AI connector type requires the `assistantEnabled` feature flag, and won't be available without it.
+
+[discrete]
+[[start-chatting]]
+== Start chatting
+
+To open the Security Assistant, press *Cmd + ;* (or *Ctrl + ;* in Windows) from anywhere in the {security-app}. This opens the *Welcome* conversation context, where you can ask general questions about {elastic-sec}.
+
+You can also chat with the Security Assistant from several areas of {elastic-sec}, and context-specific data and prompts will populate your conversation.
+
+* <<view-alert-details, Alert details>> or Event details flyout: Click *Chat* while viewing the details of an alert or event.
+* <<rules-ui-management, Rules page>>: Select one or more rules, then click the magic wand icon (🪄✨) at the top of the page next to the *Rules* title.
+* <<data-quality-dash, Data Quality dashboard>>: Select the *Incompatible fields* tab, then click *Chat*. (This is only available for fields marked red, indicating they're incompatible).
+* <<timelines-ui, Timeline>>: Select the *Security Assistant* tab.
+
+All chat history persists within your session, so you can navigate away to other areas of {elastic-sec}, then later return to ongoing conversations.
```

Comment:
I think so @joepeeples? Maybe add a `(unrelated to your kibana user session)` or something at the end so users aren't surprised if they log out and back in as another user and they still see the previous conversations?

Reply:
How about:
```suggestion
All chat history and custom quick prompts persist in local browser storage, so you can navigate away to other areas of {elastic-sec} or log in as different {kib} user, then return to ongoing conversations.
```

Reply:
Gotcha, a note callout sounds good. It's a fair amount of info now, but it's intended to prevent user confusion, and we can always revisit if needed.
```suggestion
NOTE: All chat history and custom quick prompts persist in local browser storage, allowing you to navigate away to other areas in {elastic-sec}, then return to ongoing conversations. This also means that chats persist across multiple users if they use the same browser; be sure clear any conversations that you don't want available to other users.
```